sendinblue.php

```diff
@@ -2739,14 +2830,46 @@ public function subscribeByruntimeRegister($email, $id_gender, $fname, $lname, $
             $attribute_data[] = $value_group;
             $attribute_key[] = 'GROUP_ID';
         }
-        
+        if (version_compare(_PS_VERSION_, '1.5.3.4', '>') && isset($id_shop) && !empty($email)) {
+            if ($id_shop === null) {
+                $id_shop = 1;
+            }
+            
+            $all_group_reg = array();
+            $all_group_unsubs = array();
```

Comment:
Here again, they get their actual value with `DB::getInstance(...)`. You do not need to initialize them with `array()`